DigitalLearningSolutions.Web/Views/SuperAdmin/Centres/AddCentre.cshtml

```diff
@@ -0,0 +1,173 @@
+                                            css-class="nhsuk-u-width-full"
+                                            default-option="Select a region"
+                                            select-list-options="Model.RegionNameOptions" />
+                        </div>
+                    </div>
+
+                    <div class="nhsuk-grid-row">
+                        <div class="nhsuk-grid-column-one-half">
+                            <vc:text-input asp-for="@nameof(Model.RegistrationEmail)"
+                                           label="Registration email"
+                                           populate-with-current-value="true"
+                                           type="text"
+                                           spell-check="false"
+                                           autocomplete="off"
+                                           hint-text=""
+                                           css-class="nhsuk-u-width-full"
+                                           required="false" />
+                        </div>
+                    </div>
+
+                    <div class="nhsuk-grid-row">
+                        <div class="nhsuk-grid-column-one-half">
+                            <vc:text-input asp-for="@nameof(Model.IpPrefix)"
+                                           label="IP Prefix"
+                                           populate-with-current-value="true"
+                                           type="text"
+                                           spell-check="false"
+                                           autocomplete="off"
+                                           hint-text=""
+                                           css-class="nhsuk-u-width-full"
+                                           required="false" />
+                        </div>
+                    </div>
+
+                    <div class="nhsuk-checkboxes__item">
+                        <input class="nhsuk-checkboxes__input" id="AddITSPcourses" name="AddITSPcourses" asp-for="@Model.AddITSPcourses" type="checkbox">
+                        <label class="nhsuk-label nhsuk-checkboxes__label" for="AddITSPcourses">
+                            Add ITSP courses
+                        </label>
+                    </div>
+
+                    <div class="nhsuk-checkboxes__item nhsuk-u-margin-bottom-4">
+                        <input class="nhsuk-checkboxes__input" id="ShowOnMap" name="ShowOnMap" asp-for="@Model.ShowOnMap" type="checkbox">
+                        <label class="nhsuk-label nhsuk-checkboxes__label" for="ShowOnMap">
+                            Show on find your centre
+                        </label>
+                    </div>
+
+                </div>
+            </div>
+            <button class="nhsuk-button" type="submit">
+                Save
+            </button>
+            <div class="nhsuk-back-link nhsuk-u-margin-left-1 nhsuk-u-margin-top-2">
```

Comment:
I think the NHSUKViewComponents has one of these - might be better to use it for maintainability?